lrsnashlib.h

```diff
@@ -67,7 +67,7 @@ void initFwidth(game *g);
 void updateFwidth(game *g, int col, int pos, char *str);
 
 
-long FirstTime;          /* set this to true for every new game to be solved */
+extern long FirstTime;   /* set this to true for every new game to be solved */
```

Comment:
Why defining it here at all?

Reply:
My reading of the standard is that every variable must be defined exactly once, and an `extern` declaration is not a definition.

Reply:
So you can remove it :slightly_smiling_face:

Reply:
then the variable would have internal linkage